README.md

```diff
@@ -65,12 +57,6 @@ Protect sensitive information and keep your app secure with Cloud Code’s integ
 ## Google Cloud APIs at your fingertip
 Browse available Cloud APIs, enable services, and install and learn how to integrate client libraries in your app without leaving your IDE and breaking your development flow, all with the API library browser. [Read the docs](https://cloud.google.com/code/docs/vscode/client-libraries)
 
-- Get straight to developing with Cloud Code’s simplified authentication workflow that uses your Google Cloud credentials.
```

Comment:
@vincentjocodes / @sanguinecloud - why are we removing this portion?

Reply:
This moved to https://github.com/GoogleCloudPlatform/cloud-code-vscode/pull/349/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R18